### PR TITLE
Align Path Buttons

### PR DESCRIPTION
**Pull request by tsujan**
Fixes https://github.com/lxde/pcmanfm-qt/issues/414.

The toolbar icon size is also respected.

### CODE REVIEW

**Comment by tsujan** [comment]
A screenshot with Fusion:

pathbuttons

File: src/pathbar_p.h [hunk 1]
```diff
@@ -32,10 +34,13 @@ class PathButton: public QToolButton {
     QToolButton(parent),
     pathElement_(pathElement) {
 
-      setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
+      setSizePolicy(QSizePolicy::Fixed, QSizePolicy::MinimumExpanding);
       setCheckable(true);
       setAutoExclusive(true);
       setToolButtonStyle(Qt::ToolButtonTextBesideIcon);
+      /* respect the toolbar icon size (can be set with some styles) */
+      int icnSize = qApp->style()->pixelMetric(QStyle::PM_ToolBarIconSize);
```
Comment (palinek) [inline src/pathbar_p.h]: Just a note: by the use of `QApplication::style()` you're ignoring the `QWidget::style()` (if overriden...) and also, if style is changed (or some stylesheet used..) the toolbutton widget does not adapt.

Comment (tsujan) [inline src/pathbar_p.h]: You're right. Thanks! I'll see what's best.

**Comment by tsujan** [comment]
@palinek 
It made no difference. After changing the syle, the size won't be updated until the folder is changed. But I'll replace `qApp->style()` with `style()` for now.

**Comment by tsujan** [comment]
@palinek, I think `PathBar` needs additional code to be completely style aware but the current situation is not a big deal.

**Comment by PCMan** [comment]
Merged into master. I also made it aware of style changes.
Thanks.

**Pull request closed without merging**